libpam/pam_audit.c

```diff
@@ -44,7 +44,7 @@ _pam_audit_writelog(pam_handle_t *pamh, int audit_fd, int type,
       free(buf);
   }
 
-  /* libaudit sets errno to his own negative error code. This can be
+  /* libaudit sets errno to the user's own negative error code. This can be
```

Comment:
The comment is talking about libaudit's own error code, so this edit changes the meaning.
I suggest the following wording:
`libaudit sets errno to its own negative error code.`